recipe/meta.yaml

```diff
@@ -24,16 +24,24 @@ requirements:
     - pip
   run:
     - python >=3.6
-    - toml
+    - tomli >=1.1.0
```

Comment:
Think `toml` came from [the `v2.8` `setup.py`'s `install_requires`]( https://github.com/jendrikseipp/vulture/blob/e66b203cdcf93a6fae02200db51af6e4bd2a46b3/setup.py#L51 )

Reply:
You're right, it should stay that way for compatibility purposes.